rhoas-cli/README.adoc

```diff
@@ -372,7 +347,6 @@ You can log out from the `rhoas` CLI by using the `rhoas logout` command.
 [source,shell]
 -----
 $ rhoas logout
-Successfully logged out
```

Comment:
Retain this line, and put it in an Example output block.

I know that much of the command output might seem like overkill to someone who is familiar with the commands (and requires more work to maintain), but such output is very useful to help a rookie to verify that everything is working as it should. It is also then more obvious to rookies if something _doesn't_ work as it should.